.husky/pre-commit

```diff
@@ -1,4 +1,5 @@
 #!/bin/sh
 . "$(dirname "$0")/_/husky.sh"
 
-npx lint-staged
\ No newline at end of file
+npx lint-staged
+npm run lint
```

Comment:
why do we need a separate step `npm run lint`?  `lint-staged` already does _linting_ and _formatting_ for any changed files in a commit:
https://github.com/mojaloop/mifos-core-connector/blob/ft-payer-initiated-transfer/package.json#L85C6-L85C22